pkg/apis/v2v/v1alpha1/virtualmachineimport_types.go

```diff
@@ -70,8 +61,7 @@ type VirtualMachineImportOvirtSourceVMSpec struct {
 // +k8s:openapi-gen=true
 // +optional
 type VirtualMachineImportOvirtSourceVMClusterSpec struct {
-	// +optional
-	ID *string `json:"id,omitempty"`
+	ID string `json:"id"`
```

Comment:
Why do we want to make `ID` as required?

Reply:
If the ID is required, there is no need for name field at all.
Providing ID is the recommended way, however, it isn't human readable, hence added an option to specify the name instead.

Reply:
But in this case user must specify the ID, so we don't have to have the name field at all.

Reply:
IMO, if we have he ID required, we should resign from the name field. Either way - name or ID the user will probably copy&paste the value, so I would just leave the id.

Reply:
just reading it, i realized reading Piotr's the other way around :-)
The Id was intended to remain optional. Should be fine in next update.